test/unit/modules/accounts.js

```diff
@@ -663,35 +426,35 @@ describe('account', function () {
 				});
 			});
 
-			it('should return top 10 accounts with respect to highest balance', function (done) {
-				var topAccountsBalance = _.orderBy(allAccounts, 'balance').map(function (a) {
-					return a.balance;
-				}).reverse();
-
+			it('should return top 10 accounts ordered by balance desc', function (done) {
```

Comment:
@karmacoma, which format would you suggest? Should I use `by descending balance` for specifying sort order & type throughout the file? Or keep it like 'in DESC'?